Xamarin.Forms.Platform.Android/AppCompat/FrameRenderer.cs

```diff
@@ -83,6 +83,13 @@ bool IOnTouchListener.OnTouch(AView v, MotionEvent e)
 					ScaleGestureDetectorCompat.SetQuickScaleEnabled(_scaleDetector.Value, true);
 				handled = _scaleDetector.Value.OnTouchEvent(e);
 			}
+
+			if (_gestureDetector.IsValueCreated && _gestureDetector.Value.Handle == IntPtr.Zero)
+			{
+				// This gesture detector has already been disposed, probably because it's on a cell which is going away
```

Comment:
This comment needs to be updated - "it's on a cell which is going away", perhaps change to "it's on a view which is going away".